lib/default-options.js

```diff
@@ -17,4 +17,5 @@ module.exports = {
   connectRetries: 2,
   downloadRetries: 2,
   attach: false,
+  maxDuration: 1800,
```

Comment:
Maybe default to `null` to not make this a breaking change? I'm worried that people actually have test suites which run > 30mins.